scripts/apps/search/components/Item.tsx

```diff
@@ -134,8 +138,14 @@ export class Item extends React.Component<IProps, IState> {
     }
 
     select(event) {
-        if (!this.props.item.gone) {
-            this.props.onSelect(this.props.item, event);
+        const {$timeout} = this.props.svc;
```

Comment:
Why are you using $timeout in a React component? Doesn't `setTimeout` work?

Reply:
because protractor knows it should wait for $timeouts

Reply:
Can you tell him about `setTimeout` too? :D Jokes aside, you can wait for an element in tests. The code shouldn't be coupled with test frameworks. Good locators should be all that's needed.

Reply:
Can you replace it with `setTimeout` and make the test wait for the element?